packages/account/src/Components/account-limits/account-limits.jsx

```diff
@@ -159,7 +159,11 @@ const AccountLimits = ({
                                             <Localize i18n_default_text='*Maximum account cash balance' />
                                         </AccountLimitsTableCell>
                                         <AccountLimitsTableCell align='right'>
-                                            {formatMoney(currency, account_balance, true)}
+                                            {account_balance ? (
```

Comment:
Use double exclamation operator to convert account_balance to boolean.
```suggestion
                                            {!!account_balance ? (
```

Reply:
@yauheni-kryzhyk-deriv Is it intended that we need to execute the first part of ternary expression when account_balance is a number which is more than or equal to 0, and the second part - when it's missing (null/undefined)? If yes, we have to check for 0:
```
{account_balance >= 0 ? (
```
if it's always a string, we can leave `account_balance` because an empty string (if the balance is missing) will give us false anyway.

Reply:
In that case, I'd suggest to leave a comment.